Makefile

```diff
@@ -172,36 +172,39 @@ slow_bchain_tests=$(wildcard tests/ethereum-tests/BlockchainTests/GeneralStateTe
                   # $(wildcard tests/BlockchainTests/GeneralStateTests/*/*/*_Constantinople.json)
 quick_bchain_tests=$(filter-out $(slow_bchain_tests), $(bchain_tests))
 
-bchain-test-all: $(bchain_tests:=.test)
-bchain-test: $(quick_bchain_tests:=.test)
+test-all-bchain: $(bchain_tests:=.test)
+test-bchain: $(quick_bchain_tests:=.test)
 
 tests/ethereum-tests/BlockchainTests/%.test: tests/ethereum-tests/BlockchainTests/% build
-	$(TEST) $<
+	MODE=NORMAL SCHEDULE=DEFAULT $(TEST) $< tests/templates/output-success.txt
 
 # ProofTests
 
 proof_dir:=tests/proofs/specs
 proof_tests=$(wildcard $(proof_dir)/*/*-spec.k)
 
-proof-test-all: proof-test
-proof-test: $(proof_tests:=.test)
+test-all-proof: test-proof
+test-proof: $(proof_tests:=.test)
 
 $(proof_dir)/%.test: $(proof_dir)/% build-java
-	$(TEST) $<
+	MODE=NORMAL SCHEDULE=DEFAULT $(TEST) $< tests/templates/proof-success.txt
```

Comment:
not sure if `SCHEDULE=DEFAULT` matters, but the proof spec admits the BYZANTIUM schedule.

Reply:
I think it is better to drop this, since kevm has the default value.